pkg/image/api/helper.go

```diff
@@ -128,6 +128,18 @@ func (r DockerImageReference) AsRepository() DockerImageReference {
 	return r
 }
 
+// DaemonMinimal clears defaults that Docker assumes.
```

Comment:
So we have a canonical value to compare against.  We're assuming image
references are equal in a few places, but they are not.  This is not
critical path for this pull though.

On Thu, Oct 22, 2015 at 4:36 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/image/api/helper.go
> <https://github.com/openshift/origin/pull/4819#discussion_r42802602>:
>
> > @@ -128,6 +128,18 @@ func (r DockerImageReference) AsRepository() DockerImageReference {
> >  	return r
> >  }
> >
> > +// DaemonMinimal clears defaults that Docker assumes.
>
> by breaking them if they do? what's the motivation for doing this
> minimization?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4819/files#r42802602>.
>